_docs/glider-background-and-sampling-terminology.md

```diff
@@ -20,18 +20,18 @@ summary: This page defines glider and sampling terminology used throughout the r
 
 ## Glider Types
 As of this writing, there are 3 major buoyancy driven glider types that are currently utilized by IOOS Regional Associations:
- + [Seaglider](http://www.apl.washington.edu/project/project.php?id=seaglider#:~:text=The%20AUV%20Seaglider%20is%20the,time%20via%20satellite%20data%20telemetry.): originally designed and built through a collaboration with the University of Washington's [Applied Physics Lab](http://www.apl.washington.edu/) and [School of Oceanography](http://www.ocean.washington.edu/).  The Seaglider is now manufactured by [Hydroid](https://www.hydroid.com/seaglider)
- + [Spray](http://spray.ucsd.edu/pub/rel/info/spray_description.php): originally designed by [Scripps Institution of Oceanography](https://scripps.ucsd.edu/) and [Woods Hole Oceangraphic Institution](http://www.whoi.edu/) with funding provided by the [Office of Naval Research](http://www.onr.navy.mil/), the Spray glider is now manufactured by [Bluefin Robotics](http://www.bluefinrobotics.com/products/spray-glider/).
- + [Slocum](http://www.webbresearch.com/slocumglider.aspx): designed and built by [Teledyne Webb Research Corporation](http://www.webbresearch.com/).
+ + [Seaglider](https://apl.uw.edu/project/project.php?id=seaglider#:~:text=The AUV Seaglider is the,time via satellite data telemetry.): originally designed and built through a collaboration with the University of Washington's [Applied Physics Lab](https://apl.uw.edu/) and [School of Oceanography](https://www.ocean.washington.edu/).  The Seaglider is now manufactured by [Hydroid](https://hii.com/what-we-do/capabilities/unmanned-systems/)
```

Comment:
The Hydroid used to point to a seaglider page. I could not find any mentions to seaglider in their new page. The old one redirects to https://hii.com/what-we-do/capabilities/unmanned-systems/.

Reply:
@ocefpaf Hydroid is no longer servicing Seagliders.  Change the first Seaglider link to https://apl.uw.edu/project/project.php?id=seaglider (not sure why the link there now results in highlighted text).  Just delete the sentence "The Seaglider is now manufactured by..."

Reply:
Done!